src/actions/sources.js

```diff
@@ -113,7 +120,11 @@ export function newSources(sources: Source[]) {
  */
 function loadSourceMap(generatedSource) {
   return async function({ dispatch, getState, sourceMaps }: ThunkArgs) {
-    const urls = await sourceMaps.getOriginalURLs(generatedSource);
+    let urls = null;
+    try {
+      urls = await sourceMaps.getOriginalURLs(generatedSource);
+    } catch (e) {}
```

Comment:
do we need this?

Reply:
it should be in its own PR probably. It can throw, and that breaks things